tests-functional/clients/statusgo_container.py

```diff
@@ -60,13 +60,17 @@ def __init__(self, cmd, ports=None, privileged=False, container_name_suffix=""):
             "labels": {"com.docker.compose.project": docker_project_name},
             "environment": {
                 "GOCOVERDIR": "/coverage/binary",
-                "SCAN_WAKU_FLEET": self.get_waku_fleet_scan_command(),
             },
             "volumes": {
                 coverage_path: {
                     "bind": "/coverage/binary",
                     "mode": "rw",
-                }
+                },
+                # Named volume created by docker-compose.waku.yml for wakufleetconfig.json
+                f"{docker_project_name}_wakufleetconfig": {
+                    "bind": "/usr/status-user",
+                    "mode": "rw",
+                },
```

Comment:
@at0m1x19  I think CI tests are failing because of this volume binding:
https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-7202/2/artifact/tests-functional/logs/container_test_delete_activity_center_notifications_0abc4d81.log

Effectively scanner container tries to write to CI host filesystem, which seems to be forbidden 🙁 

I remember we were fighting with this in the beginning and ended up avoiding writing files to host. Though we still copy logs, tests and coverage reports 🤔